features/step_definitions/user_steps.rb

```diff
@@ -402,3 +406,7 @@
   link = user_path(user)
   page.find(:css, %Q{a[href="#{link}?tab=activity"]}).trigger('click')
 end
+
+Then(/^I should not exist as a user$/) do
+  expect(@user).to be_nil
```

Comment:
i think, offhand, it's a scope attached to the model like -- User.with_deleted.find_by(name: "blah") @mattlindsey @tansaku